dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/interfaces/OpenTelemetryAccessor.java

```diff
@@ -60,12 +61,8 @@ protected void deactivate() {
      *         is disabled or the application has shut down.
      */
     public static OpenTelemetryInfo getOpenTelemetryInfo() {
-        try {
-            OpenTelemetryInfoFactory factory = OSGIHelpers.getService(OpenTelemetryInfoFactory.class, OpenTelemetryAccessor.class);
-            return factory.getOpenTelemetryInfo();
-        } catch (Exception e) {
-            return new ErrorOpenTelemetryInfo();
-        }
+        Optional<OpenTelemetryInfo> openTelemetryInfo = openTelemetryInfoFactoryService.call( (factory) -> {return factory.getOpenTelemetryInfo(); });
+        return openTelemetryInfo.orElse(new ErrorOpenTelemetryInfo());
```

Comment:
This will always create an instance of `ErrorOpenTelemetryInfo`.  To be the same as before it should be:
```suggestion
        return openTelemetryInfo.orElseGet(ErrorOpenTelemetryInfo::new);
```